avocado/plugins/nrun.py

```diff
@@ -95,6 +95,16 @@ def spawn_tasks(self, parallel_tasks):
             else:
                 LOG_UI.info("%s spawned and alive", identifier)
 
+    def report_results(self):
```

Comment:
I might be wrong, but it looks like this is a `StatusServer` method? What is your option about moving this to `StatusServer` class?

Reply:
I understand your point, but on the other hand, I don't see the "presentation layer" as a responsibility of the `StatusServer` class.  I think that's more the responsibility `nrun` command/plugin.  The `StatusServer` is providing the data, though.